src/components/Listing/Listing.js

```diff
@@ -14,7 +14,8 @@ const Listing = (props) => {
         <img src={imagePath} className="listing-img"/>
       </section>
       <section className="listing-button-container">
-        <button className="expand-listing-button">View Details</button>
+        <button onClick={() => props.findListing(props.listing_id)}className="listing-button">Favorite This Listing</button>
+        <button className="listing-button">View Details</button>
```

Comment:
Based on the click of a particular listing, passes that data back up to the ListingContainer to find that particular listing and pass that listing object back up to the App.